scheduler/src/cook/mesos/util.clj

```diff
@@ -451,10 +451,10 @@
    goes from guuid to a list of juuids."
   [jobs]
   (->> jobs
-     (map (fn [job]
-            (mapv #(vector (:group/uuid %) #{(:job/uuid job)}) (:group/_job job))))
-     (map (partial into {}))
-     (reduce (partial merge-with clojure.set/union))))
+       (map (fn [job]
+              (mapv #(vector (:group/uuid %) #{(:job/uuid job)}) (:group/_job job))))
+       (map (partial into {}))
+       (reduce (partial merge-with clojure.set/union))))
 
 (defn make-guuid->considerable-cotask-ids
```

Comment:
No worries, there are multiple references to `uuid` in the code: https://github.com/twosigma/Cook/search?utf8=%E2%9C%93&q=uuid&type=